.vib/victoriametrics/runtime-parameters.yaml

```diff
@@ -0,0 +1,10 @@
+# Use vmauth for accessing vminsert and vmselect
+vmauth:
+  enabled: true
+  service:
+    type: LoadBalancer
+    ports:
+      http: 80
+# Necessary for scraping metrics
+vmagent:
+  enabled: true
```

Comment:
Even thought we don't have goss tests on these "scratch" solutions, I think we should try to play with security context's values, custom configuration, etc. so we ensure the chart gets deployed when they're modified.

Reply:
But we don't have a way to verify that it works, that's the reasoning that we're setting it.

Reply:
Yes we do. For instance, if switching the `fsGroup` breaks the installation it should fail while in the "deployment" stage.

Reply:
But if it goes well, there's no way of knowing if this is actually being set